### PR TITLE
Update ios_14.md

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 1]
```diff
@@ -10,11 +10,15 @@ This guide describes Braze-related changes introduced in iOS 14 and the required
 
 For a complete list of new iOS 14 updates announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
 
+{% alert tip %}
+Apple [recently announced](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a delay to iOS 14 IDFA changes. While we still recommend upgrading to the latest version of the Braze iOS SDK (v3.27), your app will be able to continue collecting the IDFA identifier without changes to your app.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:14]:
```suggestion
Apple [recently announced](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a delay to the upcoming iOS 14 IDFA changes. While we still recommend upgrading to the latest version of the Braze iOS SDK (v3.27), your app will be able to continue collecting the IDFA identifier without changes to your app.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 2]
```diff
@@ -10,11 +10,15 @@ This guide describes Braze-related changes introduced in iOS 14 and the required
 
 For a complete list of new iOS 14 updates announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
 
+{% alert tip %}
+Apple [recently announced](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a delay to the upcoming iOS 14 IDFA changes. While we still recommend upgrading to the latest version of the Braze iOS SDK (v3.27), your app will be able to continue collecting the IDFA identifier without changes to your app.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:14]:
```suggestion
Apple [recently announced](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a delay to the upcoming iOS 14 IDFA changes. Until Apple begins to enforce this - sometime in 2021 - your app can continue to collect the IDFA identifier without changes to your app.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 3]
```diff
@@ -36,7 +40,7 @@ table td {
...
-|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|If you collect IDFA, your app must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+.<br><br>Once users upgrade to iOS 14, all apps must use the new iOS 14 APIs (Xcode 12) to prompt users for IDFA ad tracking permission. If your app does not upgrade to Xcode 12, or if a user declines the permission prompt, their IDFA value will be blank (`00000000-0000-0000-0000-000000000000`).|
+|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|In 2021, Apple will begin to enforce a permission prompt required to collect the IDFA identifier. At that time, your app must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+.<br><br>When this change goes into effect next in 2021, apps that do not use the new iOS 14 APIs will be unable to collect IDFA, and will instead collect a blank ID (`00000000-0000-0000-0000-000000000000`).|
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:43]:
```suggestion
|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|Sometime in 2021, Apple will begin to enforce a permission prompt required to collect the IDFA identifier. At that time, apps must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework to continue collecting IDFA. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+ at that time.<br><br>Apps that do not use the new iOS 14 APIs will be unable to collect IDFA, and will instead collect a blank ID (`00000000-0000-0000-0000-000000000000`) once Apple begins to enforce this change in 2021.|
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 4]
```diff
@@ -70,12 +74,12 @@ For more information on Approximate Location, see Apple's [What's New In Locatio
 
 IDFA (Identity for Advertisers) is an identifier provided by Apple used by advertising and attribution partners for cross-device tracking and is tied to an individual's Apple ID.
 
-Beginning in iOS 14, a new permission prompt (launched by the new `AppTrackingTransparency` framework) will require explicit user consent to access the IDFA. This permission to "track you across apps and websites owned by other companies" will be requested similarly to how you’d prompt users to request their location.
+Later in 2021, iOS 14 will introduce a new permission prompt (launched by the new `AppTrackingTransparency` framework) which will require explicit user consent to access the IDFA. This permission to "track you across apps and websites owned by other companies" will be requested similarly to how you’d prompt users to request their location.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:77]:
```suggestion
Later in 2021, iOS 14 will begin to enforce a new permission prompt (launched by the new `AppTrackingTransparency` framework) which will require explicit user consent to access the IDFA. This permission to "track you across apps and websites owned by other companies" will be requested similarly to how you’d prompt users to request their location.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 5]
```diff
@@ -36,7 +40,7 @@ table td {
...
-|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|If you collect IDFA, your app must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+.<br><br>Once users upgrade to iOS 14, all apps must use the new iOS 14 APIs (Xcode 12) to prompt users for IDFA ad tracking permission. If your app does not upgrade to Xcode 12, or if a user declines the permission prompt, their IDFA value will be blank (`00000000-0000-0000-0000-000000000000`).|
+|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|Sometime in 2021, Apple will begin to enforce a permission prompt required to collect the IDFA identifier. At that time, apps must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework to continue collecting IDFA. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+ at that time.<br><br>Apps that do not use the new iOS 14 APIs will be unable to collect IDFA, and will instead collect a blank ID (`00000000-0000-0000-0000-000000000000`) once Apple begins to enforce this change in 2021.|
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:43]:
```suggestion
|IDFA Ad Tracking ID| **Upgrade to Xcode 12 and iOS SDK v3.27**|Sometime in 2021, Apple will begin to require a permission prompt for the collection of the IDFA. At that time, apps must upgrade to Xcode 12 and use the new `AppTrackingTransparency` framework in order to continue collecting IDFA. If you pass IDFA to the Braze SDK you must also upgrade to v3.27.0+ at that time.<br><br>Apps that do not use the new iOS 14 APIs will be unable to collect IDFA, and will instead collect a blank ID (`00000000-0000-0000-0000-000000000000`) once Apple begins to enforce this change in 2021.|
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 6]
```diff
@@ -70,12 +74,12 @@ For more information on Approximate Location, see Apple's [What's New In Locatio
...
+Later in 2021, iOS 14 will require a new permission prompt (launched by the new `AppTrackingTransparency` framework) which will require explicit user consent to access the IDFA. This permission to "track you across apps and websites owned by other companies" will be requested similarly to how you’d prompt users to request their location.
 
 If a user does not accept the prompt, or if you do not upgrade to Xcode 12's `AppTrackingTransparency` framework, then a blank IDFA value (`00000000-0000-0000-0000-000000000000`) will be returned, and your app will not be allowed to prompt the user again.
 
 {% alert important %}
-These IDFA updates take effect once end-users upgrade their device to iOS 14. Please ensure your app is using the new `AppTransparencyFramework` and Xcode 12 even if you do not explicitly target iOS 14 support.
+These IDFA updates will take effect once end-users upgrade their device to a future iOS 14 upgrade in 2021. Once Apple announces their plans to enforce this change, please ensure your app is using the new `AppTransparencyFramework` and Xcode 12 even if you do not explicitly target iOS 14 support.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:82]:
```suggestion
Once Apple begins to enforce these IDFA updates at a future iOS 14 upgrade in 2021, your app will need to prompt users to collect IDFA. Please ensure your app is prepared to use the new `AppTransparencyFramework` and Xcode 12 even if you do not explicitly target iOS 14 support.
```

**Review by Bucimis — APPROVED** [review]
lgtm, we can address LAT-flag tracking once we have more information imo

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**